Sources/ChatToys/Utils/WebContext.swift

```diff
@@ -194,3 +198,67 @@ extension Array {
     }
 }
 
+public class MarkdownProcessor {
+    
+    public static var stringToKeyMap = [String: String]()
+    public static var uuidToURLMap = [String: String]()
```

Comment:
Can this be protected with a lock, or can the whole class be made an actor? I could imagine different threads accessing this at once and running into problems.